tests/test-multi-pow.c

```diff
@@ -0,0 +1,107 @@
+/* Test program for thread-safe dcurl */
+#include <pthread.h>
+#include "common.h"
+#include "dcurl.h"
+
+#define THREAD_MAX 10
+
+typedef struct _ducrl_item DCURL_ITEM;
```

Comment:
Use lower case form, such as `dcurl_item`. The uppercase might imply macros or constants.